docs/ballerina-to-oas/spec/spec.md

```diff
@@ -725,6 +725,26 @@ parameters:
       nullable: true
 ```
 
+The query parameter name in the schema is defaulted to the Ballerina parameter name. But this can be overridden using the `@http:Query` annotation.
```

Comment:
```suggestion
The query parameter name in the schema is defaulted to the Ballerina parameter name. But this can be overridden using `name` attribute in the `@http:Query` annotation.
```

Reply:
Fixed via [9d88386](https://github.com/ballerina-platform/openapi-tools/pull/1771/commits/9d88386ba932a402494a8e0c04687a0a1f3aeb56)